### ISSUE TITLE
API Documentation

### ISSUE THREAD

**Issue by BernhardSchlegel** [chadj/gpedal#11]
Hi Chad,

super cool project ! Where did you find the Bluetooth API spec for the trainers? Took your software for a spin yesterday :dancers: 

Best,
Bernhard.

**Comment by chadj** [comment]
I remember starting here with the web Bluetooth API's

https://developers.google.com/web/updates/2015/07/interact-with-ble-devices-on-the-web

And unfortunately, the docs I used to make sense of reading data from the bike sensors directly are no longer online.  They used to live here:

https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.characteristic.cycling_power_measurement.xml

I'm not sure if those docs exist anywhere anymore.

Here's my code for handling the cycling characteristics which might help:

https://github.com/chadj/gpedal/blob/master/src/Meter.js

**Comment by BernhardSchlegel** [comment]
Thanks so much :) Is it possible, that they just deleted the UI and that [this](https://www.bluetooth.com/wp-content/uploads/Sitecore-Media-Library/Gatt/Xml/Characteristics/org.bluetooth.characteristic.csc_measurement.xml) is the underlying document? For future use: also atttached.

[org.bluetooth.characteristic.csc_measurement.xml.txt](https://github.com/chadj/gpedal/files/5424179/org.bluetooth.characteristic.csc_measurement.xml.txt)

**Comment by chadj** [comment]
Yeah, I do believe that is the case.  The old docs were just much easier to consume.
